warden/src/assignment_computation.rs

```diff
@@ -90,11 +94,18 @@ impl AssignmentComputationImpl {
         region: Region,
     ) -> Arc<Self> {
         let s = Arc::new(Self {
-            // TODO(purujit): Initialize base_ranges from cluster manager and range_assignments from database.
+            // TODO(purujit): Initialize base_ranges from cluster manager and range_assignments incl. last version from database.
```

Comment:
the base ranges are now initialized from the universe, right?

Reply:
Yup, removed comment.